utilities/fill_locale.py

```diff
@@ -13,7 +13,7 @@
 @functools.lru_cache(maxsize=None)
 def uf_geojson(uf):
     path = config('GEOJSON_PATH')
-    filename = '{}-municipalities.json'.format(uf)
+    filename = '{}-counties.json'.format(uf)
```

Comment:
ok, I didn't know that. but for that we'd have to have the most recent
br-atlas  deployed in the server, and re-download all the files.
I think we don't have it in the server. If you want to do that, put the
repo under the /Storage directory, please.

On Fri, Oct 16, 2015 at 3:49 PM, Israel Teixeira <notifications@github.com>
wrote:

> In utilities/fill_locale.py
> <https://github.com/fccoelho/AlertaDengueCaptura/pull/11#discussion_r42276728>
> :
>
> > @@ -13,7 +13,7 @@
> >  @functools.lru_cache(maxsize=None)
> >  def uf_geojson(uf):
> >      path = config('GEOJSON_PATH')
> > -    filename = '{}-municipalities.json'.format(uf)
> > +    filename = '{}-counties.json'.format(uf)
>
> br-atlas have been updated carolinabigonha/br-atlas@9a3839c
> <https://github.com/carolinabigonha/br-atlas/commit/9a3839c29ed4c9e63fcffa589aa139aaca20fb6e>.
> Now counties are called municipalities.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fccoelho/AlertaDengueCaptura/pull/11/files#r42276728>.
>



-- 
Flávio Codeço Coelho
================
+55(21) 3799-5551
Professor
Escola de Matemática Aplicada
Fundação Getulio Vargas
Praia de Botafogo, 190 sala 312
Rio de Janeiro - RJ
22250-900
Brasil